src/status_im/ui/screens/peers_stats.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.ui.screens.peers-stats
+  (:require [quo.react-native :as quo.react-native]
```

Comment:
[Guidelines] Use `react-native.core` to call `view` and `text` components. You will be able to remove `[status-im.ui.components.react :as react]` as well.

Reply:
fixed